jboss-kie-kieserver/tests/bats/jboss-kie-kieserver.bats

```diff
@@ -617,3 +617,34 @@ teardown() {
     echo "GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE}"
     [[ "${GC_MAX_METASPACE_SIZE}" == "${KIE_SERVER_MAX_METASPACE_SIZE}" ]]
 }
+
+@test "Verify the Kafka extension" {
```

Comment:
add an invalid topic to also test the new behavior introduced.

Reply:
done